hack/api-docs/mkdocs.yml

```diff
@@ -1,10 +1,12 @@
 site_name: External Secrets Operator
 repo_url: https://github.com/external-secrets/external-secrets
 repo_name: External Secrets Operator
-site_dir: site
-docs_dir: docs
+site_dir: ../../site
+docs_dir: ../../docs
+remote_branch: gh-pages
 theme:
   name: material
+  custom_dir: ../../overrides
```

Comment:
i cross-checked it with the docs and other projects and it seems correctly configured. Could be that we have to use paths relative to the (maybe?) docs dir. Does it not change at all even if replacing `main.html` with junk?

Reply:
It seems it was some version bump that was needed. I'm testing it right now

Reply:
Indeed that was it!